cnf/build.bnd

```diff
@@ -1,23 +1,52 @@
 # Configure Repositories
--plugin.1.Central: \
+-plugin.1.R7.API: \
+	aQute.bnd.repository.maven.pom.provider.BndPomRepository; \
+		snapshotUrls=https://oss.sonatype.org/content/repositories/osgi/; \
+		releaseUrls=https://repo1.maven.org/maven2/; \
```

Comment:
Shouldn't we use the canonical url for maven central for all the plugins here?

    releaseUrl="https://repo.maven.apache.org/maven2";\

We should also set 

     readOnly=true,\

for all of the maven central plugins.